src/main/resources/model.ttl

```diff
@@ -66,6 +66,10 @@ rm:has-question rdf:type owl:ObjectProperty ;
 rm:is-member-of rdf:type owl:ObjectProperty ;
                 rdfs:subPropertyOf rm:relates-to .
 
+###  http://onto.fel.cvut.cz/ontologies/record-manager/role-group
+rm:role-group rdf:type owl:ObjectProperty ;
```

Comment:
This seems to be correct owl:ObjectProperty although above (see [1]) you declared it as owl:DataProperty ...

Reply:
done